tools/build.py

```diff
@@ -310,52 +346,102 @@ def Main():
               project_definition.name))
 
       if is_disabled:
-        disabled_packages.append(project_definition.name)
+        disabled_projects.append(project_definition.name)
       else:
         builds.append(project_definition)
 
   if not os.path.exists(options.build_directory):
     os.mkdir(options.build_directory)
 
+  undefined_projects = set(project_names)
+  for disabled_package in disabled_projects:
+    undefined_projects.remove(disabled_package)
+
+  check_configuration = set()
```

Comment:
Rename to misconfigured_builds to be consistent with the other sets here.

Reply:
ack